scripts/populate_institutions.py

```diff
@@ -137,17 +144,19 @@ def main(env):
                 'logout_url': SHIBBOLETH_SP_LOGOUT.format(encode_uri_component('https://shibboleth.nyu.edu/idp/profile/Logout')),
                 'domains': ['osf.nyu.edu'],
                 'email_domains': [],
+                'delegation_protocol': 'saml-shib',
             },
             {
                 '_id': 'okstate',
                 'name': 'Oklahoma State University',
                 'description': '<a href="http://www.library.okstate.edu/research-support/research-data-services/">OSU Library Research Data Services</a>',
                 'banner_name': 'okstate-banner.png',
                 'logo_name': 'okstate-shield.png',
-                'auth_url': None,  # https://stwcas.okstate.edu/cas/login?service=...
+                'login_url': None,  # https://stwcas.okstate.edu/cas/login?service=...
```

Comment:
Change `auth_url` to `login_url` (a few occurrences). It seems Django-OSF keeps receiving `auth_url` from new OSF merge.